source/jsonnet/northern-ireland/individual/blocks/employment/hours_worked.jsonnet

```diff
@@ -1,8 +1,41 @@
 local placeholders = import '../../../lib/placeholders.libsonnet';
 local rules = import 'rules.libsonnet';
 
-local question(title) = {
-  title: title,
+local questionTitle(isProxy, isEmployed) = (
+  local employedTitle = (
+    if isProxy then
+      {
+        text: 'In <em>{person_name_possessive}</em> main job, how many hours a week do they usually work?',
+        placeholders: [
+          placeholders.personNamePossessive,
+        ],
+      }
+    else 'In your main job, how many hours a week do you usually work?'
+  );
+  local unemployedTitle = (
+    if isProxy then
+      {
+        text: 'In <em>{person_name_possessive}</em> main job, how many hours a week did they usually work?',
+        placeholders: [
+          placeholders.personNamePossessive,
+        ],
+      }
+    else 'In your main job, how many hours a week did you usually work?'
+  );
+
+  if isEmployed then employedTitle else unemployedTitle
+);
+
+local questionDescription(isProxy, isEmployed) = (
+  if isEmployed then (
+    if isProxy then
+      ['If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>.']
+    else ['If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>.']
+  ) else []
+);
```

Comment:
This should be made consistent with questionTitle.
```jsonnet
local questionDescription(isProxy, isEmployed) = (
  local employedDescription = (
    if isProxy then
      ['If the <strong>coronavirus</strong> pandemic has affected their working hours, select the answer that best describes their <strong>current circumstances</strong>.']
    else
      ['If the <strong>coronavirus</strong> pandemic has affected your working hours, select the answer that best describes your <strong>current circumstances</strong>.']
  );

  if isEmployed then employedDescription else []
);
```

Reply:
Done, thanks!